Or5: Exception:

no backtracking
CallStack (from HasCallStack):
  error, called at Or5.hs:57:8 in main:Main

Package: ghc-internal
Module: GHC.Internal.Exception
Type: ErrorCall

HasCallStack backtrace:
  collectBacktraces, called at libraries/ghc-internal/src/GHC/Internal/Exception.hs:92:13 in ghc-internal:GHC.Internal.Exception
  toExceptionWithBacktrace, called at libraries/ghc-internal/src/GHC/Internal/Exception.hs:127:5 in ghc-internal:GHC.Internal.Exception
  error, called at Or5.hs:57:8 in main:Main

